Improve documentation

aleicazatti opened the pull request:
Add details of installation to guide a wide range of users. 
Include examples of use, output messages and error-warnings.

aleicazatti committed:
change pip command in README.md

aloctavodia commented:
For a cloned repository the name of the directory will be `13Check_RNA`, but for a downloaded repository the directory will be `13Check_RNA-master`...

aleicazatti committed:
Improve documentation

Add details of installation to guide a wide range of users. 
Include examples of use, output messages and error-warnings.

aleicazatti committed:
Improve documentation

Add details of installation to guide a wide range of users. 
Include examples of use, output messages and error-warnings.

aleicazatti committed:
Improve documentation

Add details of installation to guide a wide range of users. 
Include examples of use, output messages and error-warnings.

aleicazatti committed:
Improve documentation

Add details of installation to guide a wide range of users. 
Include examples of use, output messages and error-warnings.

aleicazatti committed:
Improve documentation

Add details of installation to guide a wide range of users. 
Include examples of use, output messages and error-warnings.

aleicazatti committed:
Improve documentation

Add details of installation to guide a wide range of users. 
Include examples of use, output messages and error-warnings

aleicazatti committed:
Improve documentation

Add details of installation to guide a wide range of users. 
Include examples of use, output messages and error-warnings

aleicazatti committed:
Improve documentation

Add details of installation to guide a wide range of users. 
Include examples of use, output messages and error-warnings.

aleicazatti committed:
Improve documentation

Add details of installation to guide a wide range of users. 
Include examples of use, output messages and error-warnings.
